plugin/src/main/java/net/codingarea/challenges/plugin/management/challenges/ChallengeLoader.java

```diff
@@ -45,6 +45,7 @@ public void enable() {
 
 		register(NoItemDamageSetting.class);
 		register(KeepInventorySetting.class);
+		register(MobGriefingSetting.class);
```

Comment:
Should be registered before KeepInventory since KeepInventory, Backpack and Enderchest are all very similar as inventory settings and should be grouped together